File: VB/SkinImages/Form1.vb
Imports Microsoft.VisualBasic
Imports System
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Data
Imports System.Drawing
Imports System.Text
Imports System.Windows.Forms

Namespace SkinImages
	Partial Public Class Form1
		Inherits Form
		Public Sub New()
			InitializeComponent()
		End Sub
		Private Sub Form1_Load(ByVal sender As Object, ByVal e As System.EventArgs) Handles MyBase.Load
			comboBoxEdit1.EditValue = defaultLookAndFeel1.LookAndFeel.SkinName
			For Each cnt As DevExpress.Skins.SkinContainer In DevExpress.Skins.SkinManager.Default.Skins
				comboBoxEdit1.Properties.Items.Add(cnt.SkinName)
			Next cnt
			GetImages()
		End Sub
		Private Sub comboBoxEdit1_SelectedValueChanged(ByVal sender As Object, ByVal e As System.EventArgs) Handles comboBoxEdit1.SelectedValueChanged
			defaultLookAndFeel1.LookAndFeel.SkinName = comboBoxEdit1.Text
			GetImages()
		End Sub

		Public Sub GetImages()
			Dim currentSkin As DevExpress.Skins.Skin
			Dim element As DevExpress.Skins.SkinElement
			Dim elementName As String
			currentSkin = DevExpress.Skins.EditorsSkins.GetSkin(defaultLookAndFeel1.LookAndFeel)
			elementName = DevExpress.Skins.EditorsSkins.SkinCheckBox
			element = currentSkin(elementName)
			Dim image As Image = element.Image.Image
			pictureEdit1.Image = image
			Dim images As DevExpress.Utils.ImageCollection = element.Image.GetImages()

			navBarControl1.SmallImages = images
			navBarControl1.Items.Clear()
			For i As Integer = 0 To images.Images.Count - 1
				Dim item As DevExpress.XtraNavBar.NavBarItem = navBarControl1.Items.Add()
				item.SmallImageIndex = i
				navBarGroup1.ItemLinks.Add(item)
			Next i
		End Sub
	End Class
End Namespace

File: CS/SkinImages/Form1.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Data;
using System.Drawing;
using System.Text;
using System.Windows.Forms;

namespace SkinImages
{
    public partial class Form1 : Form
    {
        public Form1()
        {
            InitializeComponent();
        }
        private void Form1_Load(object sender, System.EventArgs e)
        {
            comboBoxEdit1.EditValue = defaultLookAndFeel1.LookAndFeel.SkinName;
            foreach (DevExpress.Skins.SkinContainer cnt in DevExpress.Skins.SkinManager.Default.Skins)
            {
                comboBoxEdit1.Properties.Items.Add(cnt.SkinName);
            }
            GetImages();
        }
        private void comboBoxEdit1_SelectedValueChanged(object sender, System.EventArgs e)
        {
            defaultLookAndFeel1.LookAndFeel.SkinName = comboBoxEdit1.Text;
            GetImages();
        }

        public void GetImages()
        {
            DevExpress.Skins.Skin currentSkin;
            DevExpress.Skins.SkinElement element;
            string elementName;
            currentSkin = DevExpress.Skins.EditorsSkins.GetSkin(defaultLookAndFeel1.LookAndFeel);
            elementName = DevExpress.Skins.EditorsSkins.SkinCheckBox;
            element = currentSkin[elementName];
            Image image = element.Image.Image;
            pictureEdit1.Image = image;
            DevExpress.Utils.ImageCollection images = element.Image.GetImages();

            navBarControl1.SmallImages = images;
            navBarControl1.Items.Clear();
            for (int i = 0; i < images.Images.Count; i++)
            {
                DevExpress.XtraNavBar.NavBarItem item = navBarControl1.Items.Add();
                item.SmallImageIndex = i;
                navBarGroup1.ItemLinks.Add(item);
            }
        }
    }
}